### DISCUSSION TITLE
filtrowany-bezpieczny-dns/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#117] [General]
# filtrowany-bezpieczny-dns/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/filtrowany-bezpieczny-dns/?utm_content=related

**Comment by P208PUG** [comment]
Dzieki 
dziala trzeba wspomniec ze z nowym openwrt(przynajmniej snapshot) opkg nie dziala trzeba zastapic ''apk''
apk update
apk list-upgradable | cut -f 1 -d ' ' | xargs apk upgrade

apk add https-dns-proxy luci-app-https-dns-proxy

**Reply by idarek** [reply]
> Dzieki 
> dziala trzeba wspomniec ze z nowym openwrt(przynajmniej snapshot) opkg nie dziala trzeba zastapic ''apk''
> apk update
> apk list-upgradable | cut -f 1 -d ' ' | xargs apk upgrade
> 
> apk add https-dns-proxy luci-app-https-dns-proxy

OPKG mial być zastąpiony w 24.10 i to się nie stało
Narazie APT jest tylko w fazie testów więc na chwilę obecną nie będę mieszał.

**Comment by dwapir** [comment]
A co z tym ostrzeżeniem https://openwrt.org/meta/infobox/upgrade_packages_warning?

**Reply by idarek** [reply]
Osobiście się z tym, co jest na tej stronie, nie zgadzam, bo po co jest cała zabawa z OpenWrt jeżeli nie aktualizowanie pakietów?
